cmd/login/cmd.go

```diff
@@ -38,6 +39,8 @@ import (
 // #nosec G101
 var uiTokenPage string = "https://console.redhat.com/openshift/token/rosa"
 
+const oauthClientId = "ocm-cli"
```

Comment:
Does it matter to reuse the ID from the ocm-cli?

Reply:
The clientId (ocm-cli) is provided by the IT team, specifically for the CLI usage. So we can reuse this in ocm-cli as well as rosa